solidity/test/bridge/Bridge.MovingFunds.test.ts

```diff
@@ -49,8 +50,84 @@ describe("Bridge - Moving funds", () => {
 
   before(async () => {
     // eslint-disable-next-line @typescript-eslint/no-extra-semi
-    ;({ thirdParty, treasury, bank, relay, walletRegistry, Bridge, bridge } =
-      await waffle.loadFixture(fixture))
+    ;({
+      governance,
+      thirdParty,
+      treasury,
+      bank,
+      relay,
+      walletRegistry,
+      Bridge,
+      bridge,
+    } = await waffle.loadFixture(fixture))
+  })
+
+  describe("updateMovingFundsParameters", () => {
```

Comment:
`Bridge.Parameters.test.ts` please.

Reply:
Done in 49f3a5d7a6cabc18a383e4b72c7f4077cdf79499.